comp/metadata/inventoryotel/component.go

```diff
@@ -0,0 +1,17 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package inventoryotel implements a component to generate the 'datadog_agent' metadata payload for inventory.
+package inventoryotel
```

Comment:
We use to prefix payload to redapl with `inventory` back when it was only used by the inventory product and to help differentiate payloads with the same data being sent to REDAPL or the legacy endpoint.

Since OTEL is a new payload I think we can drop the prefix and simply call it `comp/metadata/otel`. WDYT ?